File: 01_Language/01_Functions/php/ctype_alnum.php
<?php

$strings = array('AbCd1zyZ9', 'foo!#$bar', '', null);
foreach ($strings as $testcase) {
    if (ctype_alnum($testcase)) {
        echo "The string $testcase consists of all letters or digits.\n";
    } else {
        echo "The string $testcase does not consist of all letters or digits.\n";
    }
}

File: 01_Language/01_Functions/python/ctype_alnum.py
# coding: utf-8

strings = ['AbCd1zyZ9', 'foo!#$bar', '']
for s in strings:
    if s.isalnum():
        print("The string " + s + " consists of all letters or digits.")
    else:
        print("The string " + s + " does not consist of all letters or digits.")
